properties/whosonfirst/whosonfirst.go

```diff
@@ -279,21 +279,27 @@ func IsAlt(f geojson.Feature) bool {
 	// 26M files and the export tools to set this property
 	// (20190821/thisisaaronland)
 
-	v := utils.StringProperty(f.Bytes(), []string{"properties.wof:alt_label"}, "")
+	// WOF admin data syntax (finalized)
+	
+	v := utils.StringProperty(f.Bytes(), []string{"properties.src:alt_label"}, "")
 
 	if v != "" {
 		return true
 	}
+	
+	// SFO syntax (initial proposal)
+	
+	w := utils.StringProperty(f.Bytes(), []string{"properties.wof:alt_label"}, "")
+
+	if w != "" {
+		return true
+	}
 
 	// we used to test that wof:parent_id wasn't -1 but that's a bad test since
 	// plenty of stuff might have a parent ID of -1 and really what we want to
 	// test is the presence of the property not the value
 	// (20190821/thisisaaronland)
 
-	if !utils.HasProperty(f.Bytes(), []string{"properties.wof:parent_id"}) {
-		return true
-	}
-
```

Comment:
I am not sure I understand the point of removing this test. Can you elaborate?

Reply:
There are more false results than valid results with this test. There are plenty of WOF records (including neighbourhoods that are not in a locality; data that is on the coast but just outside of country polygon) that have `parent_id = -1`, and for some reason they are being returned with this logic. 

It looks like the test originally was to target just alt geometries, and now that we have converted to explicate data properties, this generic and problematic test isn't needed anymore.

Reply:
The test is to ensure that a feature _has_ a `wof:parent_id` property which has always been required, regardless of its value.

If you're saying that a record with a value of `-1` is triggering this error that's a logic bug in the code. If that's the case do you have an example ID I can test against.

Reply:
ex: 890444831

Reply:
The 2nd example is common for 10s of thousands of records, like neighbourhoods that had funky Quattroshape geometries that were reset to points for their default geoms, but the QS shape was kept as an alt.

Reply:
Oh I see. Sorry, the confusion was my bad. I managed (for all this time) to think you were removing the general `wof:parent_id` check not the one is `IsAlt`. Tasteful muzak plays...